### ISSUE TITLE
(OLED42C2 ) how to enable Super Resolution in HDR PC Mode ? Its Greyed out.

### ISSUE THREAD

**Issue by NeverBeLazyG** [Maassoft/ColorControl#251]
Hey guys,

I have a 42" OLED42C2 TV. It runs on a notebook via HDMI 2.1 cable with 120Hz.I now try to activate the Super Resolution enhancer and the "super resolution" item, but it is grayed out, with the information that this is not available in PC mode.

I have another OLED, an older model, where I can also activate this in PC mode without any noticeable delay when gaming.

Since I find this enhancer very pleasant, I also want to use it on my 42" oled.

Is there a way to enable the image enhancers in HDR PC mode ? I know that "image enhancers" are not recommended in PC mode, because they increase the input lag. But I can live with it in certain cases, for example watching movies in full screen mode.

How can I use these "image enhancers" in HDR PC mode?

i installed ColorControl tool but it does not seem to work with my tv. I also could not find the right setting to unlock pc mode.

its mentioned that there should be a way to fully unlock the pc mode. but i am lost, and cant find the right setting to do so.

https://www.patreon.com/posts/77041300

Would be grateful for any tip.

Greetings

**Comment by Maassoft** [comment]
You can activate HDMI PC mode with the Expert button: Misc -> HDMI 1,2,3,4 icon -> PC.
I doubt "Super Resolution" will do a lot while in PC mode. Are there even improvements when you feed the TV a 4K signal?
Currently ColorControl does not include this setting, but I can add it to a next release.

**Comment by NeverBeLazyG** [comment]
> You can activate HDMI PC mode with the Expert button: Misc -> HDMI 1,2,3,4 icon -> PC. I doubt "Super Resolution" will do a lot while in PC mode. Are there even improvements when you feed the TV a 4K signal? Currently ColorControl does not include this setting, but I can add it to a next release.

yes the super resolution mode is usefull when you use it with normal sharpening + 25. espesially in old games with low texture quality it adds some nice artifical details. it looks more pleasent in alot old games. at least i like it alot.
